src/data/lovelace_custom_cards.ts

```diff
@@ -0,0 +1,23 @@
+export interface CustomCardEntry {
+  type: string;
+  name?: string;
+  description?: string;
+  preview?: boolean;
+}
+
+export interface CustomCardsWindow {
+  customCards?: CustomCardEntry[];
+}
+
+export const getCustomCards = () => {
```

Comment:
Instead of this, you could do this too:

```ts
const customCardsWindow = window as CustomCardsWindow
// this is at root of `lovelace_custom_cards.ts`
if ("customCards" not in customCardsWindow) {
  customCardsWindow.customCards = []
}

export const customCards = window.customCards;
```